options/posix/include/dirent.h

```diff
@@ -28,6 +28,8 @@ struct dirent {
 	char d_name[1024];
 };
 
+#define d_fileno d_ino
```

Comment:
Why is this needed? Seems like a bit of a hack

Reply:
It's a compatibility hack, used by `strace`. See [here](https://github.com/bminor/musl/blame/cfdfd5ea3ce14c6abf7fb22a531f3d99518b5a1b/include/dirent.h#L22) for what musl thought when adding it.